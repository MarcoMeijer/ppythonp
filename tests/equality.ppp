x = 1
y = "1"
if x == y:
 print("1. x == y")
if x === y:
 print("2. x === y")

a = [1, 2]
b = [1, 2]
if a === b:
 print("3. a === b")
if a ==== b:
 print("4. a === b")
a = b
if a ==== b:
 print("5. a ==== b")
