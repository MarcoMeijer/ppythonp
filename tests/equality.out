1. x == y
3. a === b
5. a ==== b
